src/theme/BlogLayout/index.js

```diff
@@ -1,40 +0,0 @@
-/**
```

Comment:
Hey @Aditya-eddy what's the reason behind removing these files are they not getting used now ?

Reply:
Also can you please check this warning 

(node:187143) [DEP_WEBPACK_COMPILATION_ASSETS] DeprecationWarning: Compilation.assets will be frozen in future, all modifications are deprecated.
BREAKING CHANGE: No more changes should happen to Compilation.assets after sealing the Compilation.
        Do changes to assets earlier, e. g. in Compilation.hooks.processAssets.
        Make sure to select an appropriate stage from Compilation.PROCESS_ASSETS_STAGE_*.
(Use `node --trace-deprecation ...` to show where the warning was created)
client (webpack 5.88.2) compiled successfully

Reply:
Hey @Hermione2408 about the deleted files I think they were used for blogs section. But I don't think there is any blog section in website. Let me look into the error!